Resources/Locale/en-US/job/job-names.ftl

```diff
@@ -73,6 +73,7 @@ JobBartender = Bartender
 JobBorg = Borg
 JobBotanist = Botanist
 JobBoxer = Boxer
+JobBrigmedic = Brigmedic
```

Comment:
Localization also uses the _starlight folder

Reply:
brigmedic was there before, its from wizden repo, i just alphabetized it
![image](https://github.com/user-attachments/assets/291cd913-cfa5-43d7-b412-abc17240637e)